jena-extras/jena-querybuilder/src/test/java/org/apache/jena/arq/querybuilder/handlers/WhereHandlerTest.java

```diff
@@ -710,27 +688,26 @@ public void testAddValueRow_pfx_collection() {
 		handler.addValueVar( query.getPrefixMapping(), v );
 		handler.addValueVar( query.getPrefixMapping(), x );
 		handler.addValueRow( query.getPrefixMapping(), Arrays.asList("<one>", "three"));
-		handler.addValueRow( query.getPrefixMapping(), Arrays.asList("<two>", "four" ));	
+		handler.addValueRow( query.getPrefixMapping(), Arrays.asList("<two>", "four" ));
 		handler.build();
-		
+
 		ElementData edat = new ElementData();
 		edat.add( v );
 		edat.add( x );
-	
-		BindingHashMap binding = new BindingHashMap();
-		binding.add( v, NodeFactory.createURI( "one" ));
-		binding.add( x, NodeFactory.createLiteral("three"));
-		edat.add( binding );
-		binding = new BindingHashMap();
-		binding.add( v, NodeFactory.createURI( "two" ));
-		binding.add( x, NodeFactory.createLiteral("four"));
-		edat.add( binding );
+        setupBindings(edat, x, v);
```

Comment:
Thought the space was wrong on this line, but looks like the rest of the file has tabs. Test file, no need to change it :+1:

Reply:
Old code had tabs.

This is why tabs are bad - display in GH is not as context sensitive as an IDE.